### ISSUE TITLE
Better Table of Contents

### ISSUE THREAD

**Issue by rcherny** [isobar-idev/code-standards#153]
I think we may need a better Table of Contents system. Thoughts include:

- the sections are big enough that the headers to a lower level might be useful
- the current side TOC obviously won't suffice
- a scrollspy type UI (#148) or
- section by section being 'sticky' (e.g. CSS stays at top in CSS section) might be useful

**Comment by rcherny** [comment]
Our current TOC is generated by JavaScript when the page is loaded. However, we could probably generate the TOC from the Markdown during the build with something like:

https://github.com/jonschlinkert/markdown-toc

**Comment by roblarsen** [comment]
Serving static HTML FTW.

**Comment by rcherny** [comment]
Yeah. Never liked that approach but our build wasn't smart enough. Now it can be.
